### ISSUE TITLE
grunt-curl encoding css

### ISSUE THREAD

**Issue by CharlJKruger** [twolfson/grunt-curl#29]
When trying to download a css file, it is encoded or something - the output looks something like:

1f8b 0800 6eb5 af54 02ff e59a 5963 1bb7
11c7 dff5 2918 ba87 e57a 7551 07a5 d48d
7b24 6dd3 2b4d 7cb4 cdd1 824b 90dc 6a77
b106 b094 e8d4 fdec 05b0 cbc3 fccf 8cfa
503f f541 0f14 e747 0083 b930 c0f3 7ca1
acd3 7e30 7cf9 e2b3 6c3c fcf8 e0e0 f8c9
a075 453d 1f3c 393e 783e 33b5 cf66 2ad7
83ef 0f06 83fe 5355 94ab 9bc1 3049 0562...

is there a parameter i need to pass or something, thanks for the awesome plugin

**Comment by twolfson** [comment]
It is most likely that is gzipped content (see #15). I am not sure why we didn't resolve this before but we should be detecting that header and gunzipping if necessary. I will take a look at patching this by the end of the weekend.

**Comment by twolfson** [comment]
Looking into this now

**Comment by twolfson** [comment]
This has been implemented, tested, and released in `2.1.0`. You will need to add the `gzip: true` option to your `src` to force `gzip` decoding:

```js
src: {
  url: 'http://my-website.com/index.js',
  gzip: true
},
dest: 'index.js'
```

**Comment by CharlJKruger** [comment]
Awesome, will test tomorrow and post the results.
